relay/worker/output_parser.go

```diff
@@ -58,7 +58,11 @@ func parseOutput(result api.ExecResult, err error, resp *messages.ExecutionRespo
 		return
 	}
 
-	resp.Status = "ok"
+	if resp.Aborted == true {
+		resp.Status = "abort"
+	} else {
+		resp.Status = "ok"
```

Comment:
I don't think that could happen given that the clauses before this one return early in the case of an error. I can clean up this logic to make it more evident.